### ISSUE TITLE
Populate database

### ISSUE THREAD

**Issue by pniedzwiedzinski** [CodeForPoznan/pah-fm#352]
I was thinking about the populate database function that we have. I've come to conclusion that when generating all data we should also create some constant test users. Before that I was using the admin user (hello@codeforpoznan.pl) for testing how the application works, but the truth is that admin account differs from driver and from passenger. I was checking how implement it and I have a solution. We could add in `backend/fleet_management/management/command/populate_database.py` 2 fixed users - 1 driver and 1 passenger. What do you guys think about it?

**Comment by pniedzwiedzinski** [comment]
In #350 all users will have the same model.

Admin should never be driver nor passenger - this cause that he is shown via API. 

That's why it's a completely different case - maybe we could add a third function for it that would be just run on the development stage.

BTW is docker also on production?

**Comment by arturtamborski** [comment]
> In #350 all users will have the same model.

Yes, you're right, but currently on develop these models are still separated.

> That's why it's a completely different case - maybe we could add a third function for it that would be just run on the development stage.

I wouldn't go with third function as it would look pretty much the same as `create_admin`, instead I'd change the behaviour (and name) of `create_admin` to `create_user` and add `User` model properties as flags, so you could specify `create_user --username [...] --is_admin TRUE --is_staff TRUE --groups X Y Z` etc. In fact I'm working on a PR for this.

Hard-coded driver and passenger would be useful for unit, manual and behave tests, just as you said.

We do not use docker on production, the setup is quite different in fact due to some limitations. Eventually I want to convert the app to a serverless setup.

**Comment by magul** [comment]
@pniedzwiedzinski 
> We could add in backend/fleet_management/management/command/populate_database.py 2 fixed users - 1 driver and 1 passenger. What do you guys think about it?

It needs to be rewritten in some part, but even right now `make populate-database` creates users (for now only drivers, but after rewrite, it will also create passengers).
At the end of its output, you are getting something like
```
Newly created users:
0. qisaienko@datsenko-chekaliuk.org
1. taras80@tryhub-babiak.org
2. lesbezborodko@ukr.net
3. eduardkabanenko@sheremeta.укр
4. perebyinisavrelii@cherednyk-vakhnii.укр
```
These users come from factories (so their attributes will be different every time), but the password for every one of them is set to `pass123` (maybe we should show that explicitly in that output?).
Doesn't it fill your needs here?

@arturtamborski 
> Admin is not a driver or passenger.

@pniedzwiedzinski  
> Admin should never be driver nor passenger - this cause that he is shown via API.

Actually admin could be a passenger or even a driver (I'm assuming, that when @OlaPAH will visit South Sudan next time, she will be using our app as a passenger and there's no reason to create a separate account for her).

> Though we could (and probably should) add --group flag to create_admin command so that Admins will have some group assigned as this breaks the app a bit.

I agree we should add `Passenger` as default group there (but would not define that as a broken app, maybe we should not allow to login when no group is set instead of failing poorly later on `403`s from APIs?)

> Moreover, passenger is a separate model not connected with User model, so there are obvious inconsistencies there.

I agree with @pniedzwiedzinski , #350 will resolve that inconsistency.

@pniedzwiedzinski 
> maybe we could add a third function for it that would be just run on the development stage.

I lost focus here. Third function for what?

@arturtamborski 
> Hard-coded driver and passenger would be useful for unit, manual and behave tests, just as you said.

* unit tests should use factories
* manual tests could be conducted using `make populate-database` output
* that's should also be possible to use factories for Behave tests (as prerequirements to Behave scenario).

**Comment by magul** [comment]
I've just checked how `ceate_admin` works and would even argue that we should not force picking country on admins. The general rule is (will be): drivers belong to one country (and therefore get projects and cars from their country only, passengers can belong to one country, but if the country field for them is unset they should appear in passenger dropdown for every driver).

It looks that we introduced country for admins just to use them to test from the drivers' perspective (and that's wrong). Also, it occurs to me that admin without Django admin access (see `ola@pah.org.pl`) doesn't make a lot of sense.
